### PR TITLE
debian: patches: add patch and update series

### PR DESCRIPTION
**Pull request by RadxaMitchell**
Add patch to r8125 driver to get led light configurations from dts and adding patch names to series. This patch is from istoreos/istoreos@52313c5
Author: jjm2473 <jjm2473@gmail.com>

### CODE REVIEW

**Comment by RadxaYuntian** [comment]
别老是关PR了，要修改直接强推你的分支就好

File: debian/patches/add-getting-led-configuration-from-dts.patch [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
++ * Bit0			LED Table V1/V2
++ * Bit1~3		    Reserved
++ * Bit4~5		    LED Blinking Duty Cycle	12.5%/ 25%/ 50%/ 75%
++ * Bit6~7		    LED Blinking Freq. 240ms/160ms/80ms/Link-Speed-Dependent
++ */
++static int
++rtl8125_led_configuration(struct rtl8125_private *tp)
++{
++        const int led_regs[] = { LEDSEL0, LEDSEL1, LEDSEL2, LEDSEL3 }; /* LEDSEL 0-3 */
++        u32 led_data[4];
```
Comment (RadxaYuntian) [inline debian/patches/add-getting-led-configuration-from-dts.patch]:
```suggestion
+        u16 led_data[4];
```
和of_property_read_u16_array的变量类型保持一致

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]

**Comment by RadxaMitchell** [comment]
> 别老是关PR了，要修改直接强推你的分支就好

不小心关掉了，*-*

**Review by RadxaYuntian — CHANGES_REQUESTED** [review]
squash

**Review by RadxaYuntian — APPROVED** [review]

**Pull request merged**